compose/Deploy/docker-compose.yaml

```diff
@@ -20,6 +20,8 @@ services:
       - 7918:7918

```

Comment:
We should probably remove the ^Ms from this file. Sorry we let them leak into it in the first place.